routes/table.js

```diff
@@ -0,0 +1,78 @@
+var express = require('express');
+var router = express.Router();
+var Table = require('../models/table');
+var tableMapper = require('../mappers/tableMapper');
+var eventMapper = require('../mappers/eventMapper');
+var isLoggedIn = require('../config/utils').isLoggedIn;
+var isAdminUser = require('../config/utils').isAdminUser;
+
+router.get('/:event_id',isLoggedIn, isAdminUser, function(req, res) {
+	eventMapper.findEventBy_event_id(req.params.event_id,function(err,result){
+
+		var eventName = result.title;
```

Comment:
if they attempt to lookup illegal/invalid tables such as table/999 and 999 doesn't exists then the server crashes. Since result is null we're attempting to get result.title. Simple condition here to redirect if result is null would avoid the crash